### PR TITLE
Copter: 4.4.0-beta1 release

### PR DESCRIPTION
**Pull request by rmackay9**
This is the Copter-4.4.0-beta1 release which is the same as the [Plane-4.4 branch](https://github.com/ArduPilot/ardupilot/tree/Plane-4.4) but with a few additional PRs added.

These additional PRs were all marked for back porting to 4.4. and can be seen in the [Copter-4.4 project](https://github.com/ArduPilot/ardupilot/projects/27)

This will also become the Rover-4.4.0-beta1 release.

I'd very much appreciate any and all feedback on the release notes to be sure I haven't missed any changes nor misrepresented the changes.

Thanks!

### CODE REVIEW

**Comment by andyp1per** [comment]
- Support 8 bi-directional dshot channels on MatekH743
- Z-axis support for baro wind estimate
- Log dynamic pressure when using wind estimate
- Matek L431 bdshot support
- CUAV Nora+ bdshot support
- In-flight FFT post filtering:
 - Allow FFT notch to be run based on filtered data
 - Provide ability to warn of motor noise at RPM frequency using FFT
 - Significantly improve in-flight FFT ability to track low frequency noise
 - Add better in-flight FFT logging
 - Support ability to read and replay IMU data for FFT analysis
- Support motor RPM takeoff check using TKOFF_RPM_MIN
- Add corner acceleration limit parameter

File: ArduCopter/ReleaseNotes.txt [hunk 1]
```diff
@@ -1,5 +1,200 @@
...
+        ii) Manual release supported (detects pilot release of gripper)
+        iii) Post release delay is configurable (see PLDP_DELAY)
+        iv) Range finder range used to protect against premature release (see PLDP_RNG_MIN)
+        v) Touchdown detection threshold is configurable (see PLDP_THRESH)
+    i) Position controller angle max adjusted inflight with CH6 Tuning knob (set TUNE=59)
+    j) Surface tracking time constant allows tuning response (see SURFTRAK_TC)
+    k) Takeoff throttle max is configurable (see TKOFF_TH_MAX)
+    l) Throttle-Gain boost increases attitude control gains when throttle high (see ATC_THR_G_BOOST)
+    m) Waypoint navigation cornering acceleration is configurable (see WPNAV_ACCEL_C)
+    n) WeatherVane option to only use during takoffs and landings (see WVANE_ENABLE)
```
Comment (IamPete1) [inline ArduCopter/ReleaseNotes.txt:82]:
```suggestion
    n) Added support for Weather Vaneing into the wing in AUTO and GUIDED (see WVANE_ENABLE)
```
I don't think the addition of the feature ever made it to the notes, maybe this is the first release.

File: ArduCopter/ReleaseNotes.txt [hunk 2]
```diff
@@ -1,5 +1,200 @@
...
+    f) Linter code checks for AP drivers and applets
+    g) Logging from scripts supports more formats
+    h) Parameters can be removed or reordered
+    i) Parameter description support (scripts must be in AP's applet or driver directory)
+    j) Rangefinder driver support
+    k) Runcam_on_arm applet starts recording when vehicle is armed
+    l) Safety switch, E-Stop and motor interlock support
+    m) Scripts can restart all scripts
+    n) Script_Controller applet supports inflight switching of active scripts
+    o) Skipcheck keywords supports skipping argument range checks
```
Comment (IamPete1) [inline ArduCopter/ReleaseNotes.txt:124]: This is probably more a developer thing.

**Comment by IamPete1** [comment]
Auto - RTL mode and do land start, possibly it was in the last release too.

- Added support for DO_LAND_START waypoints in new Auto-RTL mode.

Could mention AIS.

- Build server option to enable AIS for boat tracking

File: ArduCopter/ReleaseNotes.txt (hunk 1 above)
Comment (rmackay9) [reply ArduCopter/ReleaseNotes.txt:82]: cool, txs added.

File: ArduCopter/ReleaseNotes.txt (hunk 2 above)
Comment (rmackay9) [reply ArduCopter/ReleaseNotes.txt:124]: ok, I've zapped that line and moved the linter bit down to Developer specfic items.

**Comment by rmackay9** [comment]
@andyp1per,

Great, thanks for the feedback. I think I've included all the items in the follow-up commit.  These two were already on the list

- Z-axis support for baro wind estimate, see "a) Baro compensation using wind estimates works when climbing or descending (see BAROx_WCF_UP/DN)"
- Add corner acceleration limit paramete, see "m) Waypoint navigation cornering acceleration is configurable (see WPNAV_ACCEL_C)"

**Comment by rmackay9** [comment]
@IamPete1,

Txs again for checking this huge list.

I think the do-land-start and AUTO_RTL was included in 4.1.  We have this line from the 4.1.0-beta1 release

- d) DO_LAND_START command support for landing sequences

I'm happy to be wrong though so if I've misunderstood..

**Comment by rmackay9** [comment]
This has been double rebased against the Plane-4.4 release. https://github.com/ArduPilot/ardupilot/pull/23554

**Review by andyp1per — APPROVED** [review]

**Pull request merged**